tekton-catalog/any-sequencer/cmd/root.go

```diff
@@ -460,13 +460,80 @@ func watch(cmd *cobra.Command, args []string) {
 	go watchRun(labelSelector, tasks, failedTasksCh)
 
 	var failedTasks []string
+	var skippedTasks []string
 	for failedTask := range failedTasksCh {
 		if !contains(failedTasks, failedTask) {
 			failedTasks = append(failedTasks, failedTask)
 		}
 		if len(failedTasks) >= len(tasks) {
 			fmt.Println("All specified TaskRun(s) or Run(s) failed.")
 			exitWithStatus(skippedStatus, 1)
+		} else {
```

Comment:
This code may never been touched when there's skipped task. For example, there's 2 tasks watched(a, b), a failed and after sometime b skipped. since there's no content in failedTasksCh when b skipped, there's no chance for the logic to run.

Reply:
Yes, you are right, that's my expected, if there is any skipped tasks, should be in logic in the below changes to exit.
```
			if len(failedTasks)+len(skippedTasks) >= len(tasks) {
				fmt.Println("All specified TaskRun(s) or Run(s) failed or skipped.")
				exitWithStatus(skippedStatus, 1)
			}
```

Reply:
Yeah, great catch! Thanks @pugangxa